external/deep-object-reid/setup.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os.path as osp
+from setuptools import setup, find_packages
+
+repo_root = osp.dirname(osp.realpath(__file__))
+
+def get_requirements(filename):
+    requires = []
+    links = []
+    with open(osp.join(repo_root, filename), 'r') as f:
+        for line in f.readlines():
+            line = line.replace('\n', '')
+            if '-f http' in line:
+                links.append(line)
+            else:
+                requires.append(line)
+    return requires, links
+
+requirements, links = get_requirements('requirements.txt')
+
+print(requirements)
+print(links)
+
+setup(
+    name='torchreid_tasks',
```

Comment:
@Ilya-Krylov  @sovrasov Is this the best name that we can come up with? I mean, that d-o-r is pretty much not torchreid at all, and torchreid_tasks is even harder to tie to what is under the hood. This being said, it's not a functional remark, so we can easily leave it as is.

Reply:
My proposal is `classification_tasks`. deep_object_reid also doesn't make sense.